### PR TITLE
Cleaning UI Mess

### PR DESCRIPTION
**Pull request by sloan-seamus**
Added the Use Case diagram
Removed some un-needed files
Prettier-ed some stuff

### CODE REVIEW

File: frontend/src/app/new-claim/new-claim.component.ts [hunk 1]
```diff
@@ -52,18 +57,27 @@ export class NewClaimComponent {
   }
 
   areFieldsUntouched(): boolean {
-    return this.vin.untouched || this.color.untouched || this.model.untouched;
+    return !(this.vin.value && this.model.value && this.year.value);
```
Comment (dsifford) [inline frontend/src/app/new-claim/new-claim.component.ts:60]: If prettier did this, color me impressed.

Comment (sloan-seamus) [reply frontend/src/app/new-claim/new-claim.component.ts:60]: If you're calling me pretty, thanks, because I did it

**Review by ej3076 — APPROVED** [review]
🏅

File: frontend/src/app/new-claim/new-claim.component.ts (hunk 1 above)
Comment (dsifford) [reply frontend/src/app/new-claim/new-claim.component.ts:60]: Oh, then that's stupid. You IDIOT!

**Pull request merged**